test/testutil/vector_printer.hpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_TEST_TESTUTIL_VECTOR_PRINTER_HPP
+#define KAGOME_TEST_TESTUTIL_VECTOR_PRINTER_HPP
+
+#include <vector>
+
+namespace boost::detail::variant {
+  /**
+   * @brief Workaround for gmock unable to print std::vector<unsigned char>
```

Comment:
problem with std::vector<unsigned char> or any vector?

Reply:
There was a problem with gtest/gmock not able to print something, this was required for it to compile. Currently this file is not necessary for compilation, but can become necessary again.

Reply:
no one but you will know it and thus this file will become redundant. I suggest to remove it if it is not used

Reply:
BV knows too) but ok, I remove it.